invenio_app_ils/ui/backoffice/src/pages/ItemsSearch/ItemsSearch.js

```diff
@@ -0,0 +1,176 @@
+import React, { Component } from 'react';
+import {
+  Container,
+  Grid,
+  Segment,
+  Icon,
+  Header,
+  Button,
+  Dropdown,
+} from 'semantic-ui-react';
+import {
+  ReactSearchKit,
+  SearchBar,
+  ResultsList,
+  ResultsLoader,
+  ResultsPerPage,
+  EmptyResults,
+  Error,
+  Pagination,
+  Count,
+} from 'react-searchkit';
+import { apiConfig } from 'common/api/base';
+import { URLS } from 'common/urls';
+import { Error as IlsError } from 'common/components';
+import { item as endpoint } from 'common/api/item';
+import {
+  SearchBar as ItemsSearchBar,
+  ResultsList as ItemsResultsList,
+} from './components';
+import './ItemsSearch.scss';
+
+const resultsPerPageValues = [
```

Comment:
I think at some point we could have a separate config file for this kind of things, I have a feeling it will appear in more places, and it would be more comfortable for us to keep it in one place

Reply:
This is going to be replaced by injecting Invenio config, without harcoding values.

Reply:
As discussed, you would like to fix this in your next PR